src/digital_identity_service/receipts/receipt.item.key.response.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Validation = require('../../yoti_common/validation');
+
```

Comment:
Yeah in regards to that, I am not even sure we actually need these classes.

The methods that they correspond to are not exposed as part of the client. It seems that internally we could just use the JSON response from the method and not worry about "casting" them to some class. The only thing that class is really adding in terms of functionality is the validation aspect. Wondering if that is even useful?

In regards to your comment, yeah can add the JSdoc description and type. Although, it may not be super useful in the end given the class is not exposed.